.travis.yml

```diff
@@ -27,6 +27,10 @@ matrix:
       python: 3.7
     - env: TOXENV=py37-extra-deps
       python: 3.7
+    - env: TOXENV=py38
+      python: 3.8
+    - env: TOXENV=py37-extra-deps
```

Comment:
```suggestion
    - env: TOXENV=py38-extra-deps
```

Reply:
Actually, I think that's fine to have a single Python 3.x -extra-deps environment, to save CI time; 3.7 or 3.8-based sound fine to me.